cms/envs/aws.py

```diff
@@ -357,3 +357,20 @@
 
 PROCTORING_BACKEND_PROVIDER = AUTH_TOKENS.get("PROCTORING_BACKEND_PROVIDER", PROCTORING_BACKEND_PROVIDER)
 PROCTORING_SETTINGS = ENV_TOKENS.get("PROCTORING_SETTINGS", PROCTORING_SETTINGS)
+
+##### Third-party auth options ################################################
+if FEATURES.get('ENABLE_THIRD_PARTY_AUTH'):
+    AUTHENTICATION_BACKENDS = (
+        ENV_TOKENS.get('THIRD_PARTY_AUTH_BACKENDS', [
```

Comment:
Changes to this setting can result in the server loading arbitrary code, so for security it should not be configurable via configuration models. If you want to make an ansible setting that writes to ENV_TOKENS, that would make sense in terms of easing administration. Otherwise I'm not really sure how to share settings between the LMS and CMS - I think that's a well-known issue with the current codebase that has yet to be addressed, but is probably beyond the scope of this PR.